celi_framework/main.py

```diff
@@ -146,6 +227,20 @@ def instantiate_with_argparse_args(args: argparse.Namespace, cls: Type):
 
 
 if __name__ == "__main__":
+    tool_config_path = get_default_config_path()
+    print(f"Attempting to use config at: {tool_config_path}")
+    config = get_config()
+    if config:
+        run_celi(config)
+    else:
+        logger.error("Failed to load configuration - Check .env file and/or arguments supplied manually. Exiting...")
+
     logger.debug("Starting CELI")
     config = get_config()
     run_celi(config)
```

Comment:
This code is duplicated, so as I read it, it seems like it is just going to call run_celi twice in a row?

Either this or the section above should be deleted.